UM/Qt/QtApplication.py

```diff
@@ -551,6 +549,10 @@ def getMeshFileHandler(self) -> MeshFileHandler:
     def getWorkspaceFileHandler(self) -> WorkspaceFileHandler:
         return self._workspace_file_handler
 
+    @pyqtSlot(result = QObject)
+    def getPackageManager(self, *args):
+        return self._package_manager
+
```

Comment:
In relation to the comment above, I don't think the package manager (or in this case `self._package_manager` not being `None`) should be a required part of `QtApplication.py`.